CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ### Chore & Maintenance
 
+- `[docs]` Fix confuse filename on🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉 enableAutomock example ([#10055](https://github.com/facebook/jest/pull/10055))
```

Comment:
```suggestion
- `[docs]` Fix confuse filename on enableAutomock example ([#10055](https://github.com/facebook/jest/pull/10055))
```

Probably remove the :tada: emojis here 😅

Reply:
Hi @kenrick95 ! Good catch! :sweat_smile: 

Thanks.